src/code/z_actor.c

```diff
@@ -17,21 +18,26 @@
 #include "objects/object_bdoor/object_bdoor.h"
 
 // bss
-extern FaultClient sActorFaultClient; // 2 funcs
-extern CollisionPoly* D_801ED8B0;     // 1 func
-extern s32 D_801ED8B4;                // 2 funcs
-extern Actor* D_801ED8B8;             // 2 funcs
-extern Actor* D_801ED8BC;             // 2 funcs
-extern Actor* D_801ED8C0;             // 2 funcs
-extern Actor* D_801ED8C4;             // 2 funcs
-extern f32 D_801ED8C8;                // 2 funcs
-extern f32 sBgmEnemyDistSq;           // 2 funcs
-extern f32 D_801ED8D0;                // 2 funcs
-extern s32 D_801ED8D4;                // 2 funcs
-extern s32 D_801ED8D8;                // 2 funcs
-extern s16 D_801ED8DC;                // 2 funcs
-extern Mtx D_801ED8E0;                // 1 func
-extern Actor* D_801ED920;             // 2 funcs. 1 out of z_actor
+FaultClient sActorFaultClient; // 2 funcs
+
+CollisionPoly* D_801ED8B0; // 1 func
+s32 D_801ED8B4;            // 2 funcs
+
+Actor* sTargetableNearestActor;
+Actor* sTargetableHighestPriorityActor;
+Actor* D_801ED8C0;
+Actor* D_801ED8C4;
+
+f32 sTargetableNearestActorDistanceSq;
+f32 sBgmEnemyDistSq;
+f32 D_801ED8D0;
+s32 sTargetableHighestPriorityPriority;
```

Comment:
Is the double Priority intended?

Reply:
Sadly yes, because of this block in `Target_FindTargetableActorForCategory`
```c
                if (isNearestTargetableActor && (actor->targetPriority < sTargetableHighestPriorityPriority)) {
                    sTargetableHighestPriorityActor = actor;
                    sTargetableHighestPriorityPriority = actor->targetPriority;
                }
```

The first variable stores the address of the actor with the highest priority, while the second one tracks the priority of the actor with the highest priority. Would be fine to drop the second `Priority`?

Reply:
Would something like:
`sTargetablePrioritizedActor`
`sTargetablePrioritizedPriority`
Sound less awkward? idk